src/style-spec/expression/index.js

```diff
@@ -25,12 +25,15 @@ import type {Result} from '../util/result';
 import type {InterpolationType} from './definitions/interpolate';
 import type {PropertyValueSpecification} from '../types';
 import type {FormattedSection} from './types/formatted';
+import type Point from '@mapbox/point-geometry';
+import type {CanonicalTileID} from '../../source/tile_id';
 
 export type Feature = {
     +type: 1 | 2 | 3 | 'Unknown' | 'Point' | 'MultiPoint' | 'LineString' | 'MultiLineString' | 'Polygon' | 'MultiPolygon',
     +id?: any,
     +properties: {[_: string]: any},
-    +patterns?: {[_: string]: {"min": string, "mid": string, "max": string}}
+    +patterns?: {[_: string]: {"min": string, "mid": string, "max": string}},
+    +geometry?: Array<Array<Point>>
```

Comment:
This looks like the structure of a Polygon geometry, but wouldn’t a Point or LineString would have a different type? Or is that variation irrelevant to the type annotation here?

Reply:
It keeps the same structure as Geometry that loaded from VectorTileFeature

```
loadGeometry(): Array<Array<Point>>;
```